src/test/groovy/unit/org/pih/warehouse/requisition/RequisitionItemSpec.groovy

```diff
@@ -0,0 +1,99 @@
+package unit.org.pih.warehouse.requisition
+
+import grails.testing.gorm.DomainUnitTest
+import org.pih.warehouse.product.Product
+import org.pih.warehouse.requisition.Requisition
+import org.pih.warehouse.requisition.RequisitionItem
+import org.pih.warehouse.requisition.RequisitionItemStatus
+import org.pih.warehouse.requisition.RequisitionItemType
+import org.pih.warehouse.requisition.RequisitionStatus
+import spock.lang.Specification
+import spock.lang.Unroll
+
+@Unroll
+class RequisitionItemSpec extends Specification implements DomainUnitTest<RequisitionItem> {
+
+    void 'RequisitionItem.getDisplayStatus() should return: #requisitionItemStatus for requisition status #requisitionStatus'() {
```

Comment:
This is nitpicky but in Spock you want to have your declarations like:
```
given:
<assign your fields, mocks and stubs here>

when:
<call the method you're trying to test here>

then:
<do your asserts here>
```

or if you're just doing a single simple assert you can do: 

```
given:
<assign your fields, mocks and stubs here>

expect:
<call the method you're trying to test here and do the asserts at the same time>
```


Also, the general rule for your where: block is:

```
where:
givenFieldX | givenFieldY || expectedResultZ
```

so the fields on the left of the || is what you feed into the method in your `given:` block and the fields on the right is what you verify in the `then:` or `expect:` blocks.

Here's an example of everything together:

```
        given:
        domain.requisition = new Requisition()
        domain.requisition.status = requisitionStatus

        expect:
        domain.displayStatus == requisitionItemStatus

        where:
        requisitionStatus                  || requisitionItemStatus
        RequisitionStatus.PENDING_APPROVAL || RequisitionItemStatus.PENDING
        ...
```